src/components/app/App.js

```diff
@@ -2,21 +2,45 @@ import React, { Component } from 'react'
 import './App.css'
 
 import movieData from '../../data/movieData'
-import Movies from '../Movies/Movies'
+import Movies from '/Users/tramtram/turing_work/3mod/projects/2week/rancid-tomatillos/src/components/movies/Movies.js'
 import MoviePage from '../moviepage/MoviePage'
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      allMovies: movieData,
-      selectedMovie: ''
+      allMovies: 'movieData',
+      selectedMovie: '',
+      allMoviesLoading: true,
+      singleMovieLoading: true,
+      error: ''
     }
   }
 
+  componentDidMount = () => {
+    // console.log('Component Did Mount')
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then(response => response.json())
+      .then(data => this.setState({allMovies: data, allMoviesLoading: false}))
+      // .then(() => console.log('static data', this.state.allMovies))
+      .catch(err => this.setState({error: err}))
+  }
+
+  getSingleMovie = (id) => {
+    const url = 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/' + id
+    fetch(url)
+      .then(response => response.json())
+      // .then(data => console.log('single movie api', data))
+      .then(data => this.setState({selectedMovie: [data], singleMovieLoading: false}))
+      .catch(err => console.log(err))
```

Comment:
Is there something more useful you could do with this potential error besides console logging it? How can we make it useful to the user?

Reply:
do some error handling that will prompt the user of error as well